vault/core.go

```diff
@@ -2060,6 +2060,9 @@ func (s standardUnsealStrategy) unseal(ctx context.Context, logger log.Logger, c
 		return err
 	}
 
+	c.metricsCh = make(chan struct{})
+	go c.emitMetricsActiveNode(c.metricsCh)
```

Comment:
Is this case handled for the perf standby unseal strategy?

Reply:
No, deliberately not (hence the ActiveNode in the method name). Instead the runStandby changes will apply both for standbys and perf standbys.

Reply:
Yep. That's what I was looking to hear back. Thanks!